src/pyscaffoldext/django/extension.py

```diff
@@ -67,6 +77,17 @@ def enforce_django_options(struct, opts):
 
     return struct, opts
 
+def fix_django_settings(struct, opts):
+    old_string = f"'{opts['project']}"
+    new_string = f"'src.{opts['project']}"
```

Comment:
@FlorianWilhelm, should we hold back f-strings a little more for the sake of Python 3.5 compatibility?
(My general opinion is to wait at least until its end-of-life. Ideally I would wait until the oldest supported Ubuntu LTS, or other important distro, stop using Python 3.5)

Reply:
Good point, yes, we should hold back for version 3.3. In 4.0 I would actually break with it. I guess it will be after end-of-life of Python 3.5. Okay by you?